src/Components/CreateImageWizard/formComponents/FileSystemConfiguration.js

```diff
@@ -123,7 +123,7 @@ const FileSystemConfiguration = ({ ...props }) => {
     });
     setRows(newRows);
     setItemOrder(newOrder);
-  }, []);
+  }, [getState]);
```

Comment:
Use this pattern instead:
https://react.dev/learn/you-might-not-need-an-effect#adjusting-some-state-when-a-prop-changes

Reply:
as we discussed in the group channel, we agreed to add those dependencies
and in the future maybe we will refector the code and remove the useEffect hook were ever we can